packages/lit-dev-server/src/middleware/notfound-middleware.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import send from 'koa-send';
+
+import type Koa from 'koa';
+
+/**
+ * Creates a Koa middleware that serves a 404.html page on 404 status codes.
+ */
+export const notFoundMiddleware =
+  (staticPath: string): Koa.Middleware =>
```

Comment:
`staticPath` -> `staticRoot` so it matches the other variable name?

Reply:
Great call. Done